docs/user-manual/civiform-admin-guide/manage-questions.md

```diff
@@ -33,6 +33,21 @@ Input: 123-45-6789 Example obfuscated output: d158596dd5a6cae6fcb282832885631553
 
 Input: 123-45-6780 Example obfuscated output: 01ba4719c80b6fe911b091a7c05124b64eeece964e09c058ef8f9805daca546b
 
+### Universal questions
+
+Universal questions are questions that are intended to be used by all programs. When a question is marked as universal, it appears at the top in a separate section in the questions tab and the question bank. Any question may be set as a universal question.
+
+![](../../.gitbook/assets/universal-question.png)
```

Comment:
suggestion (non-blocking):  Since the reader may not know which view this snippet is from, I'd suggest adding a small caption above, such as "When creating a question, select the following toggle to make a question universal."

Reply:
I agree with Melanie. I think even better would be to have a screenshot that includes more of the question edit view so it looks more familiar to the user.